apps/researcher/src/app/[locale]/objects/[id]/object-lists-menu.tsx

```diff
@@ -0,0 +1,168 @@
+'use client';
+
+import {useTranslations} from 'next-intl';
+import {Fragment, useState, useEffect} from 'react';
+import {Menu, Transition} from '@headlessui/react';
+import {ChevronDownIcon} from '@heroicons/react/20/solid';
+import {CheckIcon} from '@heroicons/react/24/outline';
+import {useUser} from '@clerk/nextjs';
+import {
+  getCommunityLists,
+  addObjectToList,
+  removeObjectFromList,
+} from './object-lists-actions';
+import {SelectObjectList} from '@colonial-collections/database';
+import {useNotifications} from 'ui';
+
+interface CommunityMenuItemsProps {
+  communityId: string;
+  objectId: string;
+  userId: string;
+}
+
+function CommunityMenuItems({
+  communityId,
+  objectId,
+  userId,
+}: CommunityMenuItemsProps) {
+  const [objectLists, setObjectLists] = useState<SelectObjectList[]>([]);
+  const t = useTranslations('ObjectDetails');
+  const {addNotification} = useNotifications();
+
+  async function listClick(objectList: SelectObjectList) {
+    const objectInList = !!objectList.objects!.length;
```

Comment:
The name `objectInList`, to me, suggests that we check whether a specific object is in a list. But this line merely seems to check whether there are objects in a list. Rename to e.g. `isEmptyList` to make the meaning clearer?